### ISSUE TITLE
Gutenberg copy and paste

### ISSUE THREAD

**Issue by NewJenk** [AdvancedCustomFields/acf#75]
Copying and pasting default Gutenberg blocks copies the html, which results in a pretty decent transfer over to Google Docs/Word, but when copying ACF blocks it copies the JSON structure, it would be great to do something smart with copy and paste to replicate the default block copy and paste functionality.

**Comment by elliotcondon** [comment]
Thanks @NewJenk. I'll add this to my to-do list and test it out.

**Comment by elliotcondon** [comment]
Hi @NewJenk I've just performed some testing with this and can confirm the same functionality.
I'm tempted to consider this as a feature instead of a bug as it allows developers to copy and paste block "configuration".

You can copy and paste within the Gutenberg editor to duplicate an ACF block which is a neat feature! Modifying the "copy" logic (if this is possible) would most likely prevent this functionality which I would like to keep.

It may not be ideal for copying content out of Gutenberg, but it is definitely a great feature to copy blocks between posts/pages.

**Comment by NewJenk** [comment]
@elliotcondon I agree that being able to copy and paste blocks between posts etc is great! But could this not be done using HTML, which is then parsed to get the data needed for json? Stock Gutenberg blocks work nicely when copying and pasting between all sorts of mediums (e.g. Word, Google Docs etc), whereas copying and pasting ACF blocks to anything other than Gutenberg either doesn't display anything or displays the raw json.

**Comment by elliotcondon** [comment]
Hi @NewJenk 

All good points. The main problem we face here is that "ACF Blocks" are not normal "Gutenberg Blocks". As in, they are not 100% JS, they are dynamic. I believe this issue affects all "dynamic" blocks, as there is no way for the JS to interpret the HTML.

Something to definitely think about in the future, but not sure if I can find a solution right now.

**Comment by ChrisButterworth** [comment]
Not sure if it's tied to this but I keep getting the raw json in my clipboad when copying from an email or url field.

**Comment by flegfleg** [comment]
Any news/workarounds on this? I am in a similar situation.
